console/src/app/pages/org-settings/org-settings.component.ts

```diff
@@ -68,7 +68,7 @@ export class OrgSettingsComponent implements OnInit {
 
   ngOnInit(): void {
     this.settingsList = this.authService
-      .isAllowedMapper(this.defaultSettingsList, (setting) => setting.requiredRoles.mgmt)
+      .isAllowedMapper(this.defaultSettingsList, (setting) => setting.requiredRoles.mgmt || [])
```

Comment:
Hi @peintnermax, I've found that if we forget to add required roles to a setting that would cause an error, so if setting.requiredRoles.mgmt is undefined we'll pass an empty array to avoid that

Reply:
nice catch 👍